From 583cdb7bc4aba4a31f392975ad04d123d4a9f6f7 Mon Sep 17 00:00:00 2001
From: Nai Xia <nai.xia@gmail.com>
Date: Sun, 20 May 2012 16:10:08 +0800
Subject: [PATCH 3/3] UKSM: Missing text info in vmstat_text

---
 mm/vmstat.c |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/mm/vmstat.c b/mm/vmstat.c
index f600557..1e124c3 100644
--- a/mm/vmstat.c
+++ b/mm/vmstat.c
@@ -719,6 +719,9 @@ const char * const vmstat_text[] = {
 	"numa_other",
 #endif
 	"nr_anon_transparent_hugepages",
+#ifdef CONFIG_UKSM
+	"nr_uksm_zero_pages",
+#endif
 	"nr_dirty_threshold",
 	"nr_dirty_background_threshold",
 
-- 
1.7.9.5

